tlatools/org.lamport.tlatools/src/pcal/Tokenize.java

```diff
@@ -1486,7 +1483,7 @@ else if (nextChar == '\t')
                     { addNextChar();
                       // state = PROLOG_SPACES;
                     }
-                  else if (Misc.IsLetter(nextChar))  
+                  else if (Character.isLetter(nextChar) || '_' == nextChar)

```

Comment:
Why is Character.isLetter better than Misc.IsLetter?  As the latter is used in many places, perhaps it could be refactored to internal delegate to Character.isLetter..

Reply:
I think I'll just take the approach of overloading Misc.IsLetter function for ints.